src/advanced/components/icons/CartIcon.tsx

```diff
@@ -0,0 +1,21 @@
+interface CartIconProps {
+  className?: string;
+}
+
```

Comment:
Icon의 props 타입이 반복되는데 공통 IconProps를 만들고 필요에 따라 확장해서 쓰면 좋을 것 같습니다.
```suggestion
interface IconProps {
  className?: string;
}

interface CartIconProps extends IconProps {
  // 개별 아이콘 별로 추가적으로 필요한 속성들
}

```